src/components/html-injection/ko/htmlInjectionViewModelBinder.ts

```diff
@@ -49,7 +49,7 @@ export class HtmlInjectionViewModelBinder implements ViewModelBinder<HTMLInjecti
                 displayName: widgetDisplayName,
                 readonly: bindingContext ? bindingContext.readonly : false,
                 model: model,
-                flow: ComponentFlow.Contents,
+                flow: ComponentFlow.Block,
```

Comment:
@azaslonov I've noticed the "ComponentFlow.Contents" breaks the styling a little - 100 % width no longer makes the iframe 100 % of the parent. I thing the "Block" one is more suitable for this case, or did you change it on purpose?

Reply:
I guess I did it initially because we had a runtime widget and, hence, another wrapper element. As long as we can set both width and height, we should be good.

Reply:
(feel free to resolve this one)